test-packages/macro-fastboot-tests.test.js

```diff
@@ -0,0 +1,37 @@
+const execa = require('execa');
+const FastBoot = require('fastboot');
+
+test('FastBoot - macro-test app boots in FastBoot environment using custom app-boot', async () => {
+  jest.setTimeout(120000);
+
+  const buildProcess = execa('ember', ['build'], {
+    cwd: `${__dirname}/macro-tests`,
+  });
+
+  const buildCompletePromise = new Promise(res => {
+    buildProcess.stdout.on('data', data => {
+      const content = data.toString();
+
+      if (content.includes('"dist/"')) {
+        res(`${__dirname}/macro-tests/dist`);
+      }
+    });
+  });
+
+  const distPath = await buildCompletePromise;
+
+  // Creating application and boot application.
+  // And create an application instance by using
+  // Fastboot's `visit` API.
+  try {
+    const fastboot = new FastBoot({
+      distPath,
+    });
+
+    const resp = await fastboot.visit('/');
+    expect(resp._body).toContain('<script type="x/boundary" id="fastboot-body-start">');
+    expect(resp._body).toContain('<script type="x/boundary" id="fastboot-body-end">');
+  } catch (e) {
```

Comment:
This try/catch appears only re-throw something which is already thrown. This leads me to believe it is redundant, can we remove it?

Reply:
sure, I will fix that.

Reply:
Fixed it.